packages/clerk-js/src/ui/components/UserProfile/DeleteUserForm.tsx

```diff
@@ -36,7 +38,12 @@ export const DeleteUserForm = withCardStateProvider((props: DeleteUserFormProps)
       }
 
       await user.delete();
-      await navigateAfterSignOut();
+
+      // TODO: Investigate if we need to call `setActive` with {session: null}
```

Comment:
Do we need to do this investigation before merge?

Reply:
I was waiting for #3518 to be merged, and i will check again, but simply calling setActive would not refresh the router.

Reply:
So i investigated further, even with #3518  the issue persisted. I've identified this as a FAPI issue, because after `user.delete()` client_uat comes back with a valid timestamp instead of being `0`. This will need a separate ticket to be address and should not be part of this PR